mentoring-app/src/app/courses-page/pipes/sortByDatePipe/sort-by-date.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { CourseClass } from '../../course/models/course.models';
+
+@Pipe({
+  name: 'sortByDate'
+})
+export class SortByDatePipe implements PipeTransform {
+
+  transform(courses: CourseClass[]): any {
```

Comment:
Return value should be the same type as input value here.

Reply:
fixed